### ISSUE TITLE
Arlo GO - 30 Min Session Time

### ISSUE THREAD

**Issue by xetrok** [twrecked/hass-aarlo#148]
Hi,
Not sure where to make a request sorry!
Is it possible to add an auto refresh of the stream to avoid 30 minute timeouts for cameras?
Really appreciate the work you've put into this!

**Comment by twrecked** [comment]
Do you mean you leave a live stream running from the camera for more than 30 minutes?

**Comment by xetrok** [comment]
Yeah, I'm currently using the Arlo's as a live monitoring tool - mobile network cameras are amazing..
The glance card setup in a tab of home assistance (in addition with the motion detection videos) are a great tactical monitoring solution.

My current issue is that when live streaming (either via hass-aarlo or my.arlo portal) every stream will time out at 30 mins which makes me have to refresh/reinit the stream.

It would be great to have the functionality for the stream to re-initialize every x mins (in my case 30 mins).

**Comment by twrecked** [comment]
That was going to be my follow up question - how does my.arlo do it. 

I've never run it for that long but I'll have a think about it. Maybe I can catch timeout and automatically restart.

**Comment by xetrok** [comment]
Once reaching the 30 min time limit it times out.
Happy to share one of the cameras with you if you want to troubleshoot?
Even if you can hardcode a timer to restart the stream (if you can't catch when it dies) would be awesome!

**Comment by xetrok** [comment]
Hey mate, did you need me to do any testing for you with this one??
